platform/core.windows/apichanges.xml

```diff
@@ -84,6 +84,26 @@ is the proper place.
 
   <changes>
 
+    <change id="preferred-color-profile-spi" >
+        <api name="winsys"/>
+        <summary>Add SPI support for preferred color profile change.</summary>
+        <version major="2" minor="91"/>
```

Comment:
I would expect an increment of the module number in `manifest.mf`...

Reply:
Done. Also changed friend packages to public packages.